### ISSUE TITLE
Color is lighter than set

### ISSUE THREAD

**Issue by jfmoe** [andrew-levy/sweet-sfsymbols#9]
Hi, when I set colors for icon, it always appears lighter, whether it's white, black, or colored. 

Code:

```tsx
<SweetSFSymbol colors={['#fff']} name="chevron.backward" size={18} />
<SweetSFSymbol colors={['#fff']} name="ellipsis.circle" size={18} />
<SweetSFSymbol colors={['#fff']} name="ellipsis.circle" renderingMode="hierarchical" size={18} />
```
Results:

<img width="91" alt="image" src="https://github.com/andrew-levy/sweet-sfsymbols/assets/34602628/ba94a17d-3b40-48ad-b895-3fc04357a205">

My dependencies:

```json
"dependencies": {
    "@expo/vector-icons": "^14.0.0",
    "@react-native-async-storage/async-storage": "1.21.0",
    "@react-navigation/native": "^6.0.2",
    "@shopify/flash-list": "1.6.3",
    "expo": "~50.0.14",
    "expo-blur": "~12.9.2",
    "expo-build-properties": "~0.11.1",
    "expo-dev-client": "~3.3.11",
    "expo-document-picker": "~11.10.1",
    "expo-file-system": "~16.0.8",
    "expo-image": "~1.10.6",
    "expo-linking": "~6.2.2",
    "expo-localization": "~14.8.3",
    "expo-router": "~3.4.8",
    "expo-splash-screen": "~0.26.4",
    "expo-status-bar": "~1.11.1",
    "expo-system-ui": "~2.9.3",
    "i18next": "^23.11.1",
    "immutability-helper": "^3.1.1",
    "react": "18.2.0",
    "react-i18next": "^14.1.0",
    "react-native": "0.73.6",
    "react-native-gesture-handler": "~2.14.0",
    "react-native-ios-context-menu": "^2.5.1",
    "react-native-ios-utilities": "^4.4.5",
    "react-native-logs": "^5.1.0",
    "react-native-reanimated": "~3.6.2",
    "react-native-safe-area-context": "4.8.2",
    "react-native-screens": "~3.29.0",
    "react-native-unistyles": "^2.6.0",
    "react-native-zip-archive": "^6.1.0",
    "sweet-sfsymbols": "^0.6.0",
    "zustand": "^4.5.2"
  },
```

**Comment by andrew-levy** [comment]
@jfmoe Hey do you mind explaining the issue a little more? What exactly do you mean by "always appears lighter"? The chevron looks to be correct, are you suggesting the other two icons aren't?

**Comment by jfmoe** [comment]
> @jfmoe Hey do you mind explaining the issue a little more? What exactly do you mean by "always appears lighter"? The chevron looks to be correct, are you suggesting the other two icons aren't?

Of course. I set `#fff` for Icon, however `ellipsis.circle` looks not completely white, while `chevron.backward` is completely white. How can I set `ellipsis.circle` to completely white ?

**Comment by andrew-levy** [comment]
@jfmoe Ah okay I see the issue. It's lighter because it's being passed a `variableValue` of undefined, which gets converted to 0.0 on the swift side. By default, if no variableValue is passed in (like what you're trying to do) it should default to 1.0 (fully colored in). I'll add a fix for this! Thanks for bringing this to my attention.

**Comment by andrew-levy** [comment]
Published in `sweet-sfsymbols@0.7.0`. Can you please try this and let me know if that fixes your issue?

**Comment by jfmoe** [comment]
> Published in `sweet-sfsymbols@0.7.0`. Can you please try this and let me know if that fixes your issue?

I tried `sweet-sfsymbols@0.7.0` and it solved my issue.Thanks for your contribution！
